examples/lib/defaultHtmlStepUi.js

```diff
@@ -26,35 +26,40 @@ function DefaultHtmlStepUi(_sequencer, options) {
     step.ui =
       '\
       <div class="container">\
+      <div class="panel panel-default">\
+      <div class="panel-heading">\
+        <h3 class="panel-title">' +  
+        '<span class="toggle">' +step.name + '<span class="caret"></span></h3>\
+        </div>\
+      </h3>\
+    <div class="panel-body cal collapse in">\
     <div class="row step">\
     <form class="input-form">\
     <div class="col-md-4 details">\
     <h3>\
-    <span class = "toggle">' +step.name + ' <i class="fa fa-caret-up toggleIcon" aria-hidden="true"></i></span>' +
-    '<span class="load-spin" style="display:none;"><i class="fa fa-circle-o-notch fa-spin"></i></span>' +
+    <span class="load-spin" style="display:none;"><i class="fa fa-circle-o-notch fa-spin"></i></span>' +
     '</h3><div class="cal collapse in"><p><i>"'+
       (step.description || "") +
       '</i></p></div>\
     </div>\
-    </form>\
     <div class="col-md-8 cal collapse in step-column">\
       <div class="load" style="display:none;"><i class="fa fa-circle-o-notch fa-spin"></i></div>\
       <div class="step-image">\
-        <a class="cal collapse in"><img alt="" class="img-thumbnail step-thumbnail"/></a>\
+        <a class="cal collapse in"><img alt="" id="myimg" class="img-thumbnail step-thumbnail"/></a>\
```

Comment:
Where is the apply btn here? I cannot find it.

Reply:
Please don't give the image an id since there are many images on the page and html should not have more that one element with the same id on the same page.

Reply:
Please remove the id given to the image. Having more thsn one element with the same id is not allowed.